### DISCUSSION TITLE
Is there a way to override the methods of the FluntErrorMessages class?

### DISCUSSION THREAD

**Discussion by MateusKernDigibyte** [andrebaltieri/flunt#92] [General]
Hello, I was replacing the strings of the FluntErrorMessages class and I would also like to change how the strings in the methods of this class are assembled, would it be possible to do that?

like this:

`override FluntErrorMessages.IsFalseErrorMessage(string key) => $"This key {key} {IsFalseLocalizationErrorMessage} definitely";`

**Comment by andrebaltieri** [comment]
As they are static, you do not need to override, just set it:
`FluntErrorMessages.MessageYouWantToChange = "Your nessage"`
